Kickstarter-iOS/Views/Controllers/LiveStreamEventDetailsViewController.swift

```diff
@@ -0,0 +1,204 @@
+
+    _  = self.subscribeStackView
+      |> UIStackView.lens.axis .~ .horizontal
+      |> UIStackView.lens.alignment .~ .center
+      |> UIStackView.lens.distribution .~ .fill
+      |> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+      |> UIStackView.lens.spacing .~ Styles.grid(3)
+      |> UIStackView.lens.layoutMargins .~ .init(top: 0,
+                                                 left: Styles.grid(4),
+                                                 bottom: Styles.grid(4),
+                                                 right: Styles.grid(4))
+
+    _  = self.liveStreamTitleLabel
+      |> UILabel.lens.numberOfLines .~ 0
+      |> UILabel.lens.font %~~ { _, l in
+        l.traitCollection.isRegularRegular ? .ksr_title2() : .ksr_title2(size: 18)
+      }
+      |> UILabel.lens.textColor .~ .white
+
+    _  = self.liveStreamParagraphLabel
+      |> UILabel.lens.numberOfLines .~ 0
+      |> UILabel.lens.font %~~ { _, l in
+        l.traitCollection.isRegularRegular ? .ksr_body() : .ksr_body(size: 14)
+      }
+      |> UILabel.lens.textColor .~ .ksr_grey_500
+
+    _  = self.subscribeLabel
+      |> UILabel.lens.font .~ UIFont.ksr_headline(size: 13)
+      |> UILabel.lens.numberOfLines .~ 3
+      |> UILabel.lens.textColor .~ .white
+
+    _  = self.subscribeActivityIndicatorView
+      |> UIActivityIndicatorView.lens.activityIndicatorViewStyle .~ .white
+      |> UIActivityIndicatorView.lens.hidesWhenStopped .~ true
+
+    _  = self.subscribeButton
+      |> lightSubscribeButtonStyle
+
+    _ = [self.detailsStackView, self.subscribeStackView]
+      ||> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+      ||> UIStackView.lens.layoutMargins %~~ { _, s in
+        s.traitCollection.isRegularRegular
+          ? .init(topBottom: Styles.grid(4), leftRight: Styles.grid(12))
+          : .init(all: Styles.grid(4))
+    }
+
+    _ = self.goToProjectButton
+      |> UIButton.lens.titleColor(forState: .normal) .~ .white
+      |> liveStreamGoToProjectStyle
+  }
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.openLoginToutViewController
+      .observeValues { [weak self] _ in
+        self?.openLoginTout()
+    }
+
+    self.liveStreamTitleLabel.rac.text = self.viewModel.outputs.liveStreamTitle
+    self.liveStreamParagraphLabel.rac.text = self.viewModel.outputs.liveStreamParagraph
+    self.subscribeLabel.rac.text = self.viewModel.outputs.subscribeLabelText
+    self.subscribeButton.rac.title = self.viewModel.outputs.subscribeButtonText
+    self.subscribeButton.rac.accessibilityHint
+      = self.viewModel.outputs.subscribeButtonAccessibilityHint
+    self.subscribeButton.rac.accessibilityLabel
+      = self.viewModel.outputs.subscribeButtonAccessibilityLabel
+
+    self.viewModel.outputs.subscribeLabelHidden
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.subscribeLabel.alpha = $0 ? 0 : 1
+    }
```

Comment:
cool, if that's the case then might be worth making the outlet describe the alpha instead of the hidden state.